tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -15,6 +15,38 @@ acl/test_acl_outer_vlan.py:
     conditions:
       - "asic_type in ['cisco-8000']"
 
+acl/test_acl.py::TestBasicAcl::test_icmp_match_forwarded:
```

Comment:
@sanjair-git - Since there is workaround for this limitation (https://github.com/sonic-net/sonic-swss/pull/2178), these tests may not need to be skipped. Could you please check and update the PR?

Reply:
@tjchadaga - the fix in  (https://github.com/sonic-net/sonic-swss/pull/2178), allows us to run any of the egress ACL tests against DNX chipset. However,  due to the limitation of DNX chipset with regards to IPv6 src and dst address present in the ACL rule ( BRCM CSP #CS00012257197), 8 tests 'test_icmp_match_forwarded' are throwing errors in counters not incremented for RULE_3. Therefore, we have expect these tests to fail on DNX chipset.

Reply:
@sanmalho-git  - Is it possible to tweak the IP address to avoid this conflict with 64-bits?